api-docs/modern/modern/src/field/ComboBox.js

```diff
@@ -1,275 +1,255 @@
 /**
- * @class Ext.field.ComboBox
- * @extend Ext.field.Select
- * @xtype combobox
- * @xtype comboboxfield
- *
  * A combobox control with support for autocomplete, remote loading, and many other features.
  *
  * A ComboBox is like a combination of a traditional HTML text `<input>` field and a `<select>`
- * field; if the {@link #cfg!editable} prop is `true`, then the user is able to type freely
+ * field; if the {@link #cfg!editable} config is `true`, then the user is able to type freely
```

Comment:
I think it should read `prop` instead of `config` to be in line with react.  @mgusmano can you confirm?  @branflake2267 is there a process that runs on top of these docs that will auto convert?  I vaguely remember you mentioning that.